README.md

```diff
@@ -12,6 +12,7 @@ The NorESM documentation on Read the Docs is built from the branches in this rep
 | NorESM2.1   | [![NorESM2.1 Documentation Status](https://readthedocs.org/projects/noresm-docs/badge/?version=noresm2.1)](https://noresm-docs.readthedocs.io/en/noresm2.1/?badge=noresm2.1) | bugfix and technical release version      |
 | NorESM2.2   | [![NorESM2.2 Documentation Status](https://readthedocs.org/projects/noresm-docs/badge/?version=noresm2.2)](https://noresm-docs.readthedocs.io/en/noresm2.2/?badge=noresm2.2) | unvalidated development version           |
 | NorESM2.3   | [![NorESM2.3 Documentation Status](https://readthedocs.org/projects/noresm-docs/badge/?version=noresm2.3)](https://noresm-docs.readthedocs.io/en/noresm2.3/?badge=noresm2.3) | unvalidated development version : NorESM2.1 + FORCeS + KeyCLIM  |
+| NorESM2.5   | [![NorESM2.5 Documentation Status](https://readthedocs.org/projects/noresm-docs/badge/?version=noresm2.5)](https://noresm-docs.readthedocs.io/en/noresm2.3/?badge=noresm2.5) | development version : pre-NorESM3  |
```

Comment:
Does this work? Why is there a noresm2.3 there? Should it be:
```suggestion
| NorESM2.5   | [![NorESM2.5 Documentation Status](https://readthedocs.org/projects/noresm-docs/badge/?version=noresm2.5)](https://noresm-docs.readthedocs.io/en/noresm2.5/?badge=noresm2.5) | development version : pre-NorESM3  |
```

Reply:
Thanks, fixed typo.